source/Calamari/Commands/RunScriptCommand.cs

```diff
@@ -69,6 +69,7 @@ public override int Execute(string[] commandLineArguments)
             var conventions = new List<IConvention>
             {
                 new ContributeEnvironmentVariablesConvention(),
+                new ContributeVariablesFromJsonFileConvention(),
```

Comment:
Consider if we should/could have a test to make sure this gets included....
Might be overkill...

Reply:
I'm going to leave this out for this PR. It's a little tricky to test given how we build and run conventions at the moment. If we ever refactor how conventions are applied to commands then I think this could be a good road to go down.